packages/apollo/src/commands/service/push.ts

```diff
@@ -175,30 +203,26 @@ export default class ServicePush extends ProjectCommand {
 
     if (result.didUpdateGateway) {
       this.log(
-        `The gateway for the ${
-          result.graphName
-        } graph was updated with a new schema\n`
+        `The gateway for the ${graphString} graph was updated with a new schema, composed from the updated ${
+          result.service
```

Comment:
Now I'm a bit confused, based on my previous comments I slightly misunderstood `service` vs. `graphName`. Can we document the distinction if there's supposed to be one? In one case, they're the same, but in another, they're not. If you know the answer to my misunderstanding, some comments would be greatly appreciated. I don't expect you to have that necessarily, though, unless both codepaths are related to the work you're doing.

Reply:
Yeah, I think this could do with some clean-up. How about I take another pass at this and change around some names. I'll run the commands locally to make sure everything still prints out correctly.